dpgen/simplify/arginfo.py

```diff
@@ -25,6 +25,7 @@ def general_simplify_arginfo() -> Argument:
     doc_iter_pick_number = "The number of pick data in each iteration."
     doc_model_devi_f_trust_lo = "The lower bound of forces for the selection for the model deviation."
     doc_model_devi_f_trust_hi = "The higher bound of forces for the selection for the model deviation."
+    doc_shuffle_poscar = 'Shuffle atoms of each frame before running simulations. The purpose is to sample the element occupation of alloys.'
```

Comment:
What is "simulation" in the simplify workflow?

Reply:
Does this option take effect?

Reply:
No. It is set as false in the PARAM I get from the real case. I will remove it.